governance.md

```diff
@@ -137,6 +137,13 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 * Participate in release planning meetings and retrospectives, and burndown meetings, as needed
 * Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc. 
 * Use the above forums as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
+* Represent the SIG for the PM group:
```

Comment:
where this is in the doc is confusing, i would correlate that this is with a SIG-LEAD or whatever the new role is.  its not clear to me who in this document performs the below roles (versus the email).